lib/errorSerialization.js

```diff
@@ -0,0 +1,18 @@
+var serializeError = require('serializerr');
+var deserializeError = require('deserialize-error');
```

Comment:
For the deserializer I fully agree, it breaks also the node 0.12 build.

But the serialization is more complex than the previous serializer implementation as we need to seriliaze also other properties like `file` or something else, see for example [sass-loader error](https://github.com/webpack-contrib/sass-loader/blob/master/lib/formatSassError.js#L29)